x-pack/plugins/discover_enhanced/ui_tests/tests/value_suggestions_non_time_based.spec.ts

```diff
@@ -10,7 +10,9 @@ import { test, testData, assertionMessages } from '../fixtures';
 
 test.describe(
   'Discover app - value suggestions non-time based',
-  { tag: ['@ess', '@svlSecurity', '@svlOblt', '@svlSearch'] },
+  { tag: ['@ess', '@svlSearch', '@svlOblt'] },
+  // TODO: Update to use an ES archive with an index accessible to 'viewer'
+  // for running this test against the Security serverless project.
   () => {
     test.beforeAll(async ({ esArchiver, kbnClient, uiSettings }) => {
       await esArchiver.loadIfNeeded(testData.ES_ARCHIVES.NO_TIME_FIELD);
```

Comment:
Similar issue: `viewer` has access to limited indexes ( '.siem-signals*', '.lists-*', '.items-*') and test requires re-work.